Specify return value of main to fix:
cupl.l:220:1: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]

TODO: Declare function parameters for main and yywrap.
--- cupl.l.orig	2015-11-21 17:59:17.000000000 -0600
+++ cupl.l	2023-01-12 21:36:53.000000000 -0600
@@ -217,6 +217,7 @@
 YYSTYPE yylval;
 int	verbose;
 
+int
 main()
 {
 	int	t;
